php/sendOrderMessages.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * This example script prepares and sends information on order status changes to the Tradebyte REST API.
+ *
+ * @author Marcos Doellerer <marcos.doellerer@fatchip.de>
+ */
+
+/*
+ * API-Credentials - modify to your personal access credentials
+ *
+ * @see https://tradebyte.io/how-to/generate-rest-api-credentials-in-tb-one/
+ */
+$sApiUser = 'api-user'; // Your API username
+$sApiPassword = 'api-password'; // Your API password
+$sMerchantId = '1234'; // Your digit merchant ID
+$sChannelId = '5678'; // Your digit channel ID
+
+/*
+ * Example message
+ */
+$sNewLine = "\n";
+$sXml  = '<?xml version="1.0" encoding="utf-8"?>' . $sNewLine;
+$sXml .= '<MESSAGES_LIST>' . $sNewLine;
+$sXml .= '<MESSAGE>' . $sNewLine;
+$sXml .= '    <MESSAGE_TYPE>SHIP</MESSAGE_TYPE>' . $sNewLine;
+$sXml .= '    <TB_ORDER_ID>42</TB_ORDER_ID>' . $sNewLine;
+$sXml .= '    <TB_ORDER_ITEM_ID>52</TB_ORDER_ITEM_ID>' . $sNewLine;
+$sXml .= '    <SKU>1302</SKU>' . $sNewLine;
+$sXml .= '    <CHANNEL_SIGN>55test</CHANNEL_SIGN>' . $sNewLine;
```

Comment:
unable to parse provided XML document

```suggestion
$sXml .= '    <CHANNEL_SIGN>test</CHANNEL_SIGN>' . $sNewLine;
```